### PR TITLE
71-example-eks-clusterapi-base-cluster

### PR DESCRIPTION
**Pull request by jayanth-tjvrr**
#71 : Added a manifest file to the read me which can act as an example manifest file on which user can start off build the manifest.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -747,6 +747,85 @@ typically contains multiple related resources that together define an arbitraril
 If you make subsequent changes to the base cluster, workload clusters originally created from it
 will automatically acquire the changes.
 
+Below is an example of an manifest file with which we can create a *base cluster*. This manifest file helps in
```
Comment (cruizen) [inline README.md:750]:
```suggestion
Here is an example of a manifest file that we can use to create a *base cluster*. This manifest file helps in 
```

**Review by cruizen — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -747,6 +747,85 @@ typically contains multiple related resources that together define an arbitraril
 If you make subsequent changes to the base cluster, workload clusters originally created from it
 will automatically acquire the changes.
 
+Below is an example of an manifest file with which we can create a *base cluster*. This manifest file helps in 
+deploying an EKS cluster with machine deployment component.
```
Comment (cruizen) [inline README.md:751]:
```suggestion
deploying an EKS cluster with 'machine deployment' component from the cluster API (CAPI).
```

**Review by Rohitrajak1807 — APPROVED** [review]
apart from the typos, looks good...

**Review by ShaunakJoshi1407 — APPROVED** [review]

**Comment by cruizen** [comment]
The changes look good. Please merge.

**Pull request merged**